src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -333,6 +334,21 @@ public void TestEvaluateOptionsWithoutAliasMissingTenant()
         /// </summary>
         [Test]
         public void TestEvaluateOptionsWithoutAliasMissingRequiredOptions()
+        {
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            subject.Resource = null;
+            subject.Client = "e19f71ed-3b14-448d-9346-9eff9753646b";
+            subject.Tenant = "9f6227ee-3d14-473e-8bed-1281171ef8c9";
+            subject.Scopes = new string[] { ".default" };
+
+            subject.EvaluateOptions().Should().BeTrue();
+        }
+
+        /// <summary>
+        /// The test to evaluate options without alias missing required options.
+        /// </summary>
+        [Test]
+        public void TestEvaluateOptionsWithOverridedScopes()

```

Comment:
Method name doesn't match with the implementation. I think the above method and this method names are mismatched?

Reply:
My brain My brain was eaten by zombies.

Reply:
ROFL!! XD